canonical-websites/www.ubuntu.com
# Make list ticks orange site wide

## Done

Make list ticks orange site wide

## QA

- Pull code
- Run `./run serve`
- Navigate to [http://0.0.0.0:8001/about](http://0.0.0.0:8001/about)
- Verify that list ticks hald way down the page are orange

## Issue / Card

Related #1679
